src/main_app/db/services/owid_charts_service.py

```
@@ -7,192 +7,145 @@
+        Add a new chart.
+        """
+        chart_data = {
+            key: value for key, value in chart_data.items() if value is not None and hasattr(OwidChartRecord, key)
+        }
+        try:
+            chart = OwidChartRecord(**chart_data)
+            self.session.add(chart)
+            self.session.commit()
+            self.session.refresh(chart)
+            return chart
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)